docs/docsite/rst/dev_guide/developing_collections_path.rst

```diff
@@ -0,0 +1,160 @@
+.. _developing_collections_path:
+
+*******************************
+Ansible collection creator path
+*******************************
+
+The guide's purpose and overview
+================================
+
+.. note::
+
+  If you are totally unfamiliar with the collection technology, first take a look at the :ref:`Using Ansible collections guide<collections>`.
+
+Ansible collections are a distribution format for Ansible content that can include playbooks, roles, modules, and plugins.
+A typical collection addresses a set of related use cases. For example, the ``community.dns`` collection includes modules and plugins to work with DNS.
+
+You can install collections made by others or share yours with the community through a distribution server such as Ansible Galaxy. Certified collections can be published to the Red Hat Automation Hub, a part of the Red Hat Ansible Automation Platform.
+
+Creating and sharing collections is a great way of contribution to the Ansible project.
+
+The Ansible community package consists of ``ansible-core``, which, among other core components, includes the ``ansible.builtin`` collection maintained by the Core team, and a set of collections maintained by the community.
+
+The purpose of this guide is to give your as a (potential) content creator a consistent overview of the Ansible collection creator journey from an idea for the first module/role to having your collection included in the Ansible community package. The :ref:`Collection development guidelines section<developing_collections>` provides references to more detailed aspect-specific documents.
+
+Each of the following guide sub-sections will reflect one milestone in the path.
+
+.. _examine_existing_content:
+
+Examine currently available solutions
+=====================================
+
+If you have an idea for a new role or module/plugin, there is no need to reinvent the wheel if there is already a sufficient solution that solve your automation issue.
+
+Therefore, first examine the currently available content including:
+
+* `Ansible builtin modules and plugins <https://docs.ansible.com/ansible/latest/collections/ansible/builtin/index.html>`_
+* :ref:`Ansible package collection index<all_modules_and_plugins>`
+* `Ansible Galaxy <https://galaxy.ansible.com/>`_
+* `Ansible Automation Hub <https://www.ansible.com/products/automation-hub>`_ if you have the Ansible Automation Platform subscription
+
+In case the solutions you found are not fully sufficient or have flaws, consider improving them rather than creating your own.
```

Comment:
```suggestion
In case the solutions you found are not fully sufficient or have flaws, consider improving them rather than creating your own. Each collection includes information on where to create issues for that collection to propose your enhancement ideas.
```